packages/visx-brush/src/BaseBrush.tsx

```diff
@@ -111,6 +127,145 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
     }
   }
 
+  componentDidMount() {
+    window.addEventListener('mouseup', this.handleMouseUp, false);
```

Comment:
should the listeners be added / removed conditional on whether `props.useWindowMoveEvents === true`?